kvdb/postgres/db.go

```diff
@@ -0,0 +1,233 @@
+// +build kvdb_postgres
+
+package postgres
```

Comment:
Is anything in this file _actually_ postgres specific? Would replacing the database conn string w/ MYSQL just work out of the box?

Reply:
No it wouldn't because some of the SQL is db engine specific. Maybe it could be generalized, but this could have a performance impact. Also some special operations like listing tables don't have a generic SQL statement for it afaik